### PR TITLE
Added Arabic Language Localization

### PR DESCRIPTION
**Pull request by MaherJendoubi**

### CODE REVIEW

**Comment by MaherJendoubi** [comment]
@thiennn Please review

File: src/SimplCommerce.WebHost/appsettings.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿{
   "ConnectionStrings": {
-    "DefaultConnection": "Server=.\\sqlexpress;Database=SimplCommerce;uid=sa;pwd=sa;Trusted_Connection=True;MultipleActiveResultSets=true"
+    "DefaultConnection": "Server=(localdb)\\MSSQLLocalDB;Database=SimplCommerce;uid=sa;pwd=sa;Trusted_Connection=True;MultipleActiveResultSets=true"
```
Comment (YuriiPovkh) [inline src/SimplCommerce.WebHost/appsettings.json]: We need change config file?

Comment (MaherJendoubi) [inline src/SimplCommerce.WebHost/appsettings.json]: @YuriiPovkh I forgot to exclude it from my commit. I am sorry for that. We do not need to change config file.

Comment (thiennn) [inline src/SimplCommerce.WebHost/appsettings.json]: @MaherJendoubi thanks, but we also need to support PostgreSQL. Could you please help to make it? or I will do

Comment (MaherJendoubi) [inline src/SimplCommerce.WebHost/appsettings.json]: @thiennn I will do it

Comment (MaherJendoubi) [inline src/SimplCommerce.WebHost/appsettings.json]: @thiennn BTW Arabic language is written from right to left.

Comment (MaherJendoubi) [inline src/SimplCommerce.WebHost/appsettings.json]: @thiennn UI Should be adapated for RTL.

**Comment by MaherJendoubi** [comment]
@thiennn Please review.

**Comment by thiennn** [comment]
thank you

**Pull request merged**